base/challenge-skeleton/healthcheck/exploit/doit.py

```diff
@@ -14,6 +14,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from pwn import *
+import pwnlib
+
+def handle_pow(r):
+    r.recvuntil('./pow.py solve ')
+    challenge = r.recvline().strip()
```

Comment:
Don't you need to send the challenge to proc?

Reply:
oops, I think I tested this without enabling the proof of work ;)